elide-async/src/main/java/com/yahoo/elide/async/service/AsyncQueryThread.java

```diff
@@ -73,12 +74,10 @@ else if (queryObj.getQueryType().equals(QueryType.GRAPHQL_V1_0)) {
             if (response == null) {
                 throw new NoHttpResponseException("Response for request returned as null");
             }
-
             // Create AsyncQueryResult entry for AsyncQuery and
             // add queryResult object to query object
             asyncQueryDao.createAsyncQueryResult(response.getResponseCode(), response.getBody(),
```

Comment:
we can create the async query result object here and pass it to the new method described in the comments for DAO.